test/PProf.jl

```diff
@@ -26,10 +26,13 @@ end
 @testset "export basic profile" begin
     Profile.clear()
 
-    @profile for i in 1:10000
-        # Profile compilation
-        @eval foo(x,y) = x * y + x / y
-        @eval foo($i,3)
+    while Profile.len_data() == 0
+        @profile for i in 1:10000
+            # Profile compilation
+            foo_sym = Symbol("foo$i)")
+            @eval $foo_sym(x,y) = x * y + x / y
+            @eval $foo_sym($i,3)
+        end
```

Comment:
This change silences the warnings we were getting about redefining the same function name in a loop